tests_login_with_email_address/test_visibility_of_elements.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from assertpy import assert_that, soft_assertions
+
+
+def test_visibility(page):
+    with soft_assertions():
+
+        # page.goto("https://login.pracuj.pl/")
+
+        locators = page.locator("[data-test=\"input-email\"]")
```

Comment:
`locator` is not descriptive enough. The general rule for naming conventions should be that variable is elf descriptive to for this one should be something like `email_input_loc`